.github/workflows/test.yml

```diff
@@ -96,3 +96,53 @@ jobs:
         run: |
           call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
           bash -c "ci/scripts/build_example.sh $(pwd)/example"
+  ubuntu-meson:
```

Comment:
I'm happy to try and create a matrix of these jobs as required. There weren't any matrix jobs today so I didn't want to introduce that here. Can do as a follow up or here if desired